Gruntfile.js

```diff
@@ -420,8 +420,10 @@ module.exports = function (grunt) {
             '*.html',
             'images/{,*/}*.{webp}',
             'styles/fonts/{,*/}*.*',
+            'styles/gridset/{,*/}*.*',
```

Comment:
Yes, all the css files are required. Take a look at here:
https://github.com/clearbooks/labs-frontend/pull/42/files#diff-163ff155e1adbc8b4356afb57952d4ffR3